tests/wpunit/AssetsTest.php

```diff
@@ -332,4 +363,69 @@ public function should_allow_setting_dependencies_with_a_callable(): void {
 			ob_get_clean()
 		);
 	}
+
+	/**
+	 * Evaluates if a script and style have been registered.
+	 */
+	protected function existence_assertions( $test_slug_prefix ) {
+		$this->assertTrue( Assets::init()->exists( $test_slug_prefix . '-script' ) );
+		$this->assertTrue( Assets::init()->exists( $test_slug_prefix . '-style' ) );
+		$this->assertTrue( wp_script_is( $test_slug_prefix . '-script', 'registered' ) );
+		$this->assertTrue( wp_style_is( $test_slug_prefix . '-style', 'registered' ) );
+		$this->assertEquals( $test_slug_prefix . '-script', Assets::init()->get( $test_slug_prefix . '-script' )->get_slug() );
+		$this->assertEquals( $test_slug_prefix . '-style', Assets::init()->get( $test_slug_prefix. '-style' )->get_slug() );
+	}
+
+	/**
+	 * Asserts that the minified version of a script or style is found.
+	 *
+	 * @param string $slug_prefix
+	 * @param bool   $is_js
+	 * @param bool   $has_min
+	 * @param bool   $has_only_min
+	 */
+	protected function assert_minified_found( $slug_prefix, $is_js = true, $has_min = true, $has_only_min = false ) {
+		$asset = Assets::init()->get( $slug_prefix . '-' . ( $is_js ? 'script' : 'style' ) );
+
+		$url = get_site_url() . '/wp-content/plugins/assets/tests/_data/' . ( $is_js ? 'js' : 'css' ) . '/' . $slug_prefix;
+
+		$urls = [];
+
+		uopz_redefine( 'SCRIPT_DEBUG', false );
```

Comment:
This redefinition, and the following ones together with the `uopz_undefine` call, must be moved to a method that will wrap the redefinition to avoid some redefinition or undefinition from leaking to following tests.